README.md

```diff
@@ -1,2 +1,53 @@
-# reviewapps-deploy-status
-A Github Action that tests the deploy status of a Heroku Review App
+# Review Apps deployment status
+A Github Action that tests the deploy status of a Heroku Review App.
+
+## Usage:
+* Include the action in the workflow
+    ```yaml
+    name: Review App Test
+
+    on:
+        pull_request:
+            branches:
+            - master
+
+    jobs:
+    review-app-test:
+
+        runs-on: ubuntu-latest
+        
+        steps:
+        - uses: actions/checkout@v1
+        - name: Run review-app test
+        uses: #TODO: After publishing the action
+        env:
+            GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+            interval: 10 # optional in seconds
+            accepted_responses: 200 # default
+    ```
+
+> Note: Work flow should include `pull_request` event.
+
+* `GITHUB_TOKEN` is required to communicate with Github Deployment API. Default token provided by [Github](https://help.github.com/en/articles/virtual-environments-for-github-actions#github_token-secret) can be used.
+* Input description:  
+    All the inputs are optional. If needed, must be provided from the workflow.
+
+    | Name  | Description  | Default   | 
+    |---|---|---|
+    | interval | Wait for this amount of seconds before retrying the build check  | 10  |
+    | accepted_responses | status_codes to be excluded | 200  |
```

Comment:
You say `accepted_responses` and then `to be excluded`. Either the parameter name is bad, or the description. Things need to be consistent!